src/verify/VerificationStartScreen.tsx

```diff
@@ -255,6 +267,7 @@ function VerificationStartScreen({
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+    justifyContent: 'center',
```

Comment:
i think this is a duplicate style?

Reply:
This is so the loading arrow displays in the center without it I was seeing it display at the top after changes.

Reply:
Looks like it duplicated after #3064 was merged to main. Looks like we both saw the issue. Fixed in f08a67eb24edd1e351d63b020b160a50f416288f!